keyboards/torn/keymaps/kinesish/README.md

```diff
@@ -0,0 +1,6 @@
+# A Custom Keymap for [Torn](https://github.com/rtitmuss/torn)
```

Comment:
Oops, one thing actually - could you rename this file to lowercase?

Reply:
sure thing!

Reply:
and done.